services/app-api/prisma/migrations/20231031204348_migrate_questions_to_contracts/migration.sql

```diff
@@ -0,0 +1,17 @@
+BEGIN;
+
+ALTER TABLE "Question" ADD COLUMN "contractID" TEXT;
+
+-- AddForeignKey
+ALTER TABLE "Question" ADD CONSTRAINT "Question_contractID_fkey" FOREIGN KEY ("contractID") REFERENCES "ContractTable"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
+
+-- migrate data over
+UPDATE "Question" SET "contractID"="pkgID";
```

Comment:
This is where we copy all the pkgIDs into contractIDs

The way this works is that we don't add the NOT NULL constraint until after we copy this over. 
then we can safely drop the pkgID column